web/controllers/session_controller.ex

```diff
@@ -1,38 +1,22 @@
 defmodule Yggdrasil.SessionController do
   use Yggdrasil.Web, :controller
 
-  alias Yggdrasil.User
-  alias Yggdrasil.Session
+  require Logger
 
-  # go to client if logged in
-  plug Yggdrasil.Plug.RedirectIfAuthenticated when action in [:create, :new]
-  # replaces empty strings with nil I believe
-  # recommended to use mainly with html forms
-  # also checks to see if it's there I think?!
-  plug :scrub_params, "user" when action in [:create]
+  alias Yggdrasil.Session
 
-  # not sure why this can't be index...
-  def new(conn, _params) do
-    render conn, changeset: User.create_changeset(%User{})
-  end
+  @invalid_username %{title: "invalid username/password", 
+                      detail: "invalid username/password provided"}
```

Comment:
Excellent point, both could be displayed to the user will fix that up along with the two other typos you have seen.